docs/website/docs/developers/debugging/sanitizers.md

```diff
@@ -39,6 +39,65 @@ Enabling ASan in the IREE build is a simple matter of setting the
 cmake -DIREE_ENABLE_ASAN=ON ...
 ```
 
+#### Linking to the dynamic ASan runtime
+
+You may want to use ASan when using the python bindings.
+One way to achieve this is to build Python with ASan or whatever executable is
```

Comment:
.. to build Python (or whatever executable that is going to use IREE as a shared library) with Asan.